runtime/Cpp/runtime/src/RuntimeMetaData.cpp

```diff
@@ -9,30 +9,34 @@ using namespace antlr4;
 
 const std::string RuntimeMetaData::VERSION = "4.9.3";
 
-std::string RuntimeMetaData::getRuntimeVersion() {
-  return VERSION;
-}
+std::string RuntimeMetaData::getRuntimeVersion() { return VERSION; }
 
 void RuntimeMetaData::checkVersion(const std::string &generatingToolVersion, const std::string &compileTimeVersion) {
   std::string runtimeVersion = VERSION;
   bool runtimeConflictsWithGeneratingTool = false;
   bool runtimeConflictsWithCompileTimeTool = false;
 
   if (generatingToolVersion != "") {
-    runtimeConflictsWithGeneratingTool = runtimeVersion != generatingToolVersion
-      && getMajorMinorVersion(runtimeVersion) != getMajorMinorVersion(generatingToolVersion);
+    runtimeConflictsWithGeneratingTool =
+        runtimeVersion != generatingToolVersion &&
+        getMajorMinorVersion(runtimeVersion) != getMajorMinorVersion(generatingToolVersion);
```

Comment:
Here I would like to keep the original form: if a boolean expression gets too long for one line split it into groups (based on the operator precedence) so that a boolean operator starts a line, indicating so the grouping of the subexpressions (applies also to `if` statements).